api/cases/caseRouter.js

```diff
@@ -192,6 +192,16 @@ router.get('/', Cache.checkCache, (req, res) => {
  *        description: 'case not found'
  */
 
+router.get('/unapproved-cases', (req, res) => {
+  Cases.findByStatus('review')
+    .then((cases) => {
+      res.status(200).json(cases);
+    })
+    .catch((err) => {
+      res.status(500).json(err.message);
+    });
+});
+
 router.get('/:id', (req, res) => {
```

Comment:
We learned it's probably good practice to put all the dynamic ids at the end of files!